genai-perf/docs/multi_modal.md

```diff
@@ -45,23 +46,120 @@ docker run --runtime nvidia --gpus all \
+
+Run GenAI-Perf to generate/send all three modalities to the server
 ```bash
+export NVIDIA_API_KEY=your_api_key
+
 genai-perf profile \
-    -m llava-hf/llava-v1.6-mistral-7b-hf \
-    --tokenizer llava-hf/llava-v1.6-mistral-7b-hf \
+    -m microsoft/phi-4-multimodal-instruct \
+    -u https://integrate.api.nvidia.com \
     --service-kind openai \
-    --endpoint-type vision \
+    --endpoint-type multimodal \
+    --synthetic-input-tokens-mean 10 \
+    --output-tokens-mean 10 \
+    --image-width-mean 50 \
+    --image-height-mean 50 \
+    --audio-length-mean 3 \
+    --audio-depths 16 32 \
+    --audio-sample-rates 16 44.1 48 \
+    --audio-num-channels 2 \
+    --audio-format wav \
+    --streaming \
+    --header "Authorization: Bearer '${NVIDIA_API_KEY}'"
+```
+
+Console output will have the following result table
+
+```bash
+                          NVIDIA GenAI-Perf | Multi-Modal Metrics
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┓
+┃                         Statistic ┃    avg ┃    min ┃    max ┃    p99 ┃    p90 ┃    p75 ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━┩
+│          Time To First Token (ms) │ 284.94 │ 228.38 │ 562.57 │ 371.15 │ 325.76 │ 295.26 │
+│         Time To Second Token (ms) │  10.02 │   8.79 │  10.94 │  10.89 │  10.59 │  10.32 │
+│              Request Latency (ms) │ 346.80 │ 251.96 │ 662.96 │ 463.03 │ 404.79 │ 382.18 │
+│          Inter Token Latency (ms) │   9.56 │   0.00 │  15.76 │  14.22 │  11.05 │  10.81 │
+│   Output Sequence Length (tokens) │   7.43 │   1.00 │  14.00 │  14.00 │  12.00 │  10.00 │
+│    Input Sequence Length (tokens) │  10.11 │  10.00 │  11.00 │  11.00 │  11.00 │  10.00 │
+│ Output Token Throughput (per sec) │  20.96 │    N/A │    N/A │    N/A │    N/A │    N/A │
+│      Request Throughput (per sec) │   2.82 │    N/A │    N/A │    N/A │    N/A │    N/A │
+│             Request Count (count) │ 101.00 │    N/A │    N/A │    N/A │    N/A │    N/A │
+└───────────────────────────────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+```
+
+
+## Generating Multi-Modal Requests with GenAI-Perf
+
+Currently, you can send multi-modal contents with GenAI-Perf using the following two approaches:
+1. The synthetic data generation approach, where GenAI-Perf generates the multi-modal data for you.
+2. The Bring Your Own Data (BYOD) approach, where you provide GenAI-Perf with the data to send.
+
+
+### Approach 1: Synthetic Multi-Modal Data Generation
+
+GenAI-Perf can generate synthetic data of three modalities (text, image, and audio)
+using the modality-specific parameters provide by the user through CLI.
+Checkout [CLI Input Options](../README.md#input-options) for a complete list of parameters
+that you can tweak for different modalities.
+
+```bash
+genai-perf profile \
+    -m <multimodal_model> \
+    --service-kind openai \
+    --endpoint-type multimodal \
+    # audio parameters
```

Comment:
These blocks are meant to be copy and paste able.
Will these comments break the copy and paste functionality?

Reply:
sounds good. thanks for clarifying